streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -86,7 +86,7 @@ class Checkpoint(ssc: StreamingContext, val checkpointTime: Time)
     }
 
     // Add Yarn proxy filter specific configurations to the recovered SparkConf
-    val filter = "org.apache.hadoop.yarn.server.webproxy.amfilter.AmIpFilter"
+    val filter = "org.apache.spark.deploy.yarn.AmIpFilter"
```

Comment:
ditto

Reply:
the class defined in the `yarn` module is not visible in the `streaming` module

Reply:
Yes, we can't use `classOf[AmIpFilter].getName` here due to dependency issues.